docs/api/apiv3/paths/days_non_working_date.yml

```diff
@@ -0,0 +1,133 @@
+# /api/v3/days/non_working/{date}
+---
+get:
+  summary: View a non-working day
+  operationId: view_non_working_day
+  tags:
+    - Work Schedule
+    - Non working days
+  description: |-
+    Returns the non-working day information for a given date.
+  parameters:
+    - name: date
+      in: path
+      required: true
+      description: |-
+        The date of the non-working day to view in ISO 8601 format.
+      schema:
+        type: string
+        format: date
+      example: '2022-05-06'
+  responses:
+    '200':
+      description: OK
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/non_working_day_model.yml'
+    '400':
+      description: Returned if the client sends invalid request parameters.
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+    '404':
```

Comment:
Maybe. That's the kind of things I was not sure about.

@ulferts do you think some of the days endpoints can be public, or they at least need to be logged-in?

Reply:
This endpoint should be publicly available, so no 403.
If we restrict to logged in users, that means frontend will not be able to display elements of public projects properly (Gantt chart, calendars, etc.) because the week days and non-working days information can not be fetched.